apostrophecms/a3-docs
# change image

## Summary

*Summarize the changes briefly, including which issue/ticket this resolves. If it closes an existing Github issue, include "Closes #[issue number]"*
A user on Discord pointed out a confusing image in our a3 documentation that looks like it is a hold-over from a2. This PR changes that image for a new one. Closes PRO-3829.

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly
1. Build documentation
2. Navigate to `/guide/pages.html#connecting-pages-with-page-tree-navigation`

## What kind of change does this PR introduce?
*(Check at least one)*


## Make sure the PR fulfills these requirements:


If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
